### ISSUE TITLE
Can the editor be more scalable?

### ISSUE THREAD

**Issue by abraxas86** [murilopolese/kano-pixel-kit-pixel32#17]
It's really hard to follow along with tutorials and stuff when I can't even shrink the window down without the editor saying it's too small.

I was wondering if there were plans to make it more scalable in the future?  Is it possible to connect to the device via SSH or something instead of having to use the web app?

**Comment by murilopolese** [comment]
Unfortunately I don't have plans to continue development of this project, but you can use other software to interact with the PixelKit such as my current project: https://github.com/arduino/lab-micropython-editor
